azure-pipelines-20230430.yml

```diff
@@ -214,7 +214,7 @@ stages:
             displayName: Top 100 long-running testcases
       - job: UT_FT_3
         displayName: UT spark-datasource Java Tests & DDL
-        timeoutInMinutes: '90'
+        timeoutInMinutes: '120'
```

Comment:
let's make sure that colstats support for test table, and lowering the test timeout is tracked somewhere.